TROUBLESHOOTING.md

```diff
@@ -0,0 +1,151 @@
+# Troubleshooting
+
+Use this guide to troubleshoot issues and resolve errors that may occur when notarial-api or notarial-worker is deployed.
+
+Connect to the database:
+```sh
+kubectl run -it --rm --env PGPASSWORD='<PASSWORD>' --env PAGER= --image=postgres:16  --restart=Never postgres-client -- psql -h <ENDPOINT_URL> -U master -d notarial
+```
+
+Replace PASSWORD with the password for the database, ENDPOINT_URL with the endpoint URL for the database.  
+
+## pgboss
+
+[pgboss](https://github.com/timgit/pg-boss) is used to manage queueing jobs. On application start, pgboss will automatically create necessary tables in the database.
+
+### Jobs table
+The jobs table `pgboss.job` is where all the current jobs are stored. Jobs will remain here, until they are completed or failed. Then they will move to `pgboss.archive`
+
+The jobs table has the following columns:
+
+```
+    Column    |            Type             | Collation | Nullable |           Default           
+--------------+-----------------------------+-----------+----------+-----------------------------
+ id           | uuid                        |           | not null | gen_random_uuid()
+ name         | text                        |           | not null | 
+ priority     | integer                     |           | not null | 0
+ data         | jsonb                       |           |          | 
+ state        | pgboss.job_state            |           | not null | 'created'::pgboss.job_state
+ retrylimit   | integer                     |           | not null | 0
+ retrycount   | integer                     |           | not null | 0
+ retrydelay   | integer                     |           | not null | 0
+ retrybackoff | boolean                     |           | not null | false
+ startafter   | timestamp with time zone    |           | not null | now()
+ startedon    | timestamp with time zone    |           |          | 
+ singletonkey | text                        |           |          | 
+ singletonon  | timestamp without time zone |           |          | 
+ expirein     | interval                    |           | not null | '00:15:00'::interval
+ createdon    | timestamp with time zone    |           | not null | now()
+ completedon  | timestamp with time zone    |           |          | 
+ keepuntil    | timestamp with time zone    |           | not null | now() + '14 days'::interval
+ on_complete  | boolean                     |           | not null | false
+ output       | jsonb                       |           |          | 
+```
+
+Columns/values to note are
+- `name`: the name of the job. This will be "submission" for forms-worker
```

Comment:
Should this be related to the job types for notarial-api, e.g SES_PROCESS, NOTIFY_PROCESS etc.?